tests/unit_threaded/ut/integration.d

```diff
@@ -82,3 +82,10 @@ import unit_threaded.integration;
         shouldEqualLines("lines.txt", ["foo", "toto"]);
     }
 }
+
+@safe unittest {
```

Comment:
Thanks. Given that this is specifically for Windows, this should be versioned. I think that's why CI is failing.

Reply:
Should writeFile append `/n` at the end? Test doesn't account for that yet.

Reply:
IMO writeFile should write the exact string without \n at the end